python-checks/src/test/resources/pandasModifyInPlace.py

```diff
@@ -0,0 +1,95 @@
+def non_compliant_1():
+    import pandas as pd
+
+    df = pd.DataFrame({'A': [3, 2, 1], 'B': ['x', 'y', 'z']})
+    df2 = pd.DataFrame({'A': [6, 5, 4], 'C': ['a', 'b', 'c']})
```

Comment:
Why do we need `df2` here? It doesn't seem used. Same in the other methods.

Reply:
Indeed it was a bit unnecessary. I improved the test cases in a5d9fcc0fc25598cadd33cfd97e2b8d4e3a972f8.